### DISCUSSION TITLE
0.4.x where is model whitelist？

### DISCUSSION THREAD

**Discussion by nijisakai** [open-webui/open-webui#7133] [General]
hundreds of users. i can't find whitelist in new version.

**Comment by tjbck** [comment]
I'd strongly recommend reading the full changelogs/release logs before updating to the latest version, especially for the major 0.x version bump (e.g. 0.3 -> 0.4).

Related: https://github.com/open-webui/open-webui/discussions/7099

<img width="897" alt="image" src="https://github.com/user-attachments/assets/62fdd0c6-0432-4668-8cae-d0a4948a379c">

<img width="1465" alt="image" src="https://github.com/user-attachments/assets/0feb7ccb-fef2-42d2-988d-e04ded0a2523">

<img width="1463" alt="image" src="https://github.com/user-attachments/assets/225783a8-d189-4be5-845e-e2e88325a506">

**Reply by nijisakai** [reply]
Thank you very much!

**Comment by mowtschan** [comment]
What about ordering? There is no way now to sort the model with drag&drop or anyhow.
Another disscussion #7086
